utilities/docker/geppetto-frontend/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM java:7

```

Comment:
@jrmartin is there an easy way to not duplicate the docker files but rather having a parameter that lets switch behaviour so that it can be used both in frontend with whatever branch or by default using development